Vagrantfile

```diff
@@ -59,6 +59,12 @@ EOM
         apt-get install --yes nodejs
         npm install --global npm@latest
         npm install --global bower
-
+        npm install -g grunt-cli
+        npm install --global grunt
     SHELL
+    config.vm.provision "shell", run: 'always',inline: <<-SHELL2
+    cd /vagrant/mbofui
+    npm install
+    grunt --gruntfile Gruntfile-vagrant.js
+    SHELL2
```

Comment:
Indent the lines between the `config.vm.provision` line and this closing `SHELL2` line.  That will improve readability and show more clearly what's being done in this provisioning step.